packages/components/src/internal/components/domainproperties/assay/AssayPropertiesInput.tsx

```diff
@@ -305,11 +305,11 @@ export class AutoCopyDataInput extends React.PureComponent<InputProps, AutoCopyD
 
         return (
             <AssayPropertiesInput
-                label="Auto-Copy Data to Study"
+                label="Auto-Link Data to Study"
                 helpTipBody={
                     <>
                         <p>
-                            When new runs are imported, automatically copy their data rows to the specified target
+                            When new runs are imported, automatically link their data rows to the specified target
                             study. Only rows that include subject and visit/date information will be copied.
```

Comment:
One more "copied" reference at the end of this help tip info. I think changing it as well to "linked" makes sense.

Reply:
Oop, thanks! Pitfall of ctrl + f for 'copy' is missing 'copied/copies' :)